### ISSUE TITLE
Docker 2.5.0.1 crashes on start up Windows 10 v20H2

### ISSUE THREAD

**Issue by prime23-gareth** [docker/for-win#9471]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [x] I have uploaded Diagnostics
  - Diagnostics ID: 7A65DEF7-D853-4A4A-AA22-E626EB7D505C/20201111085413

### Expected behavior

Docker starts up with no errors.

### Actual behavior

Docker does not start up.

### Information

The Docker Desktop Service is running. I have tried restarting it and running the Docker application again. Same issue.

System.InvalidOperationException:
Failed to deploy distro docker-desktop to C:\Users\XXXXX\AppData\Local\Docker\wsl\distro: exit code: -1
 stdout: The service cannot be started, either because it is disabled or because it has no enabled devices associated with it.

  - Windows Version: 10  20H2
  - Docker Desktop Version: 2.5.0.1
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: NO

### Steps to reproduce the behavior

Docker set to start automatically on Windows boot up. Crashes once logged into Windows.

**Comment by msrs** [comment]
Try running the following command from powershell
`wsl --unregister docker-desktop-data`

**Comment by prime23-gareth** [comment]
Hi, thanks for suggestion but it gives a similar error:
```
PS C:\Users\Gareth> wsl --unregister docker-desktop-data
The service cannot be started, either because it is disabled or because it has no enabled devices associated with it.
```

**Comment by prime23-gareth** [comment]
I have now managed to resolve this issue finally! The LxssManager had become disabled and this was stopping the WSL system running correctly. Access was denied to restart it. To resolve this I uninstalled the Windows Subsystem for Linux, rebooted and re-installed. (I also uninstalled/re-installed Docker and the Windows Container feature for good measure).
After that it all started working again.
